tests/test_image_fit.py

```
@@ -0,0 +1,404 @@
+    ‏`disk_fits` בסוכן **אינו משתנה כאן** — הוא נטען מהקובץ כמו שהוא.
+    אותו tiny11 שנחסם על ה-NVMe עובר את אותה בדיקה בדיוק אחרי שהמחיצה
+    המורחבת כווצה במדויק ב-`shrink_bytes`, ולא בייט יותר. אילו החישוב
+    בקליטה והבדיקה בשחזור היו נגזרים משני כללים, המספר הזה היה מפספס
+    לכאן או לכאן — ואז או שהאימג' עדיין נחסם, או שהוא נכתב וגולש.
+    """
+    missing = shrink_bytes(LAB_TINY11, NVME_256)
+    shrunk = shrunk_by(missing)
+
+    before = fits_box(tmp_path / "before", f"echo {NVME_256}", image=LAB_TINY11)
+    after = fits_box(tmp_path / "after", f"echo {NVME_256}", image=shrunk)
+    for (run, path, prelude), want in ((before, "rc=1"), (after, "rc=0")):
```

Ruff (0.16.3)

[warning] Loop control variable `run` not used within loop body

Rename unused `run` to `_run`

(B007)